python/tvm/relay/prelude.py

```diff
@@ -515,3 +923,20 @@ def __init__(self, mod=None):
         self.define_tree_size()
 
         self.define_iterate()
+
+        self.define_tensor_adt()
+        self.define_tensor_take()
+        self.define_tensor_add_one()
+        self.define_tensor_concat()
+        self.define_tensor_array()
+        self.define_tensor_array_read()
+        self.define_tensor_array_size()
+        self.define_tensor_array_write()
+        self.define_tensor_array_stack()
+        self.define_tensor_array_unstack_tensor1()
+        self.define_tensor_array_unstack_tensor2()
+        self.define_tensor_array_scatter()
+        # TODO(wweic): Gather fails in PartialEvaluate
+        # self.define_tensor_array_gather()
```

Comment:
then TensorArrayGatherV3 is not working correctly?

Reply:
Yes, you are correct. There is currently an issue in PartialEvaluate for `GatherV3`.